code/datums/body_archive.dm

```diff
@@ -10,6 +10,8 @@ var/list/body_archives = list()
 	var/key
 	var/rank
 
+	var/datum/mind/mind//keeping a pointer to the source mind for tracking purposes
```

Comment:
The actual compiler seems to deal with this just fine, but the syntax highlighting doesn't like it